src/main/webapp/app/shared/markdown-editor/select-with-search/select-with-search.component.html

```diff
@@ -0,0 +1,9 @@
+<div #dropdown ngbDropdown class="d-inline-block" (openChange)="$event ? handleMenuOpen() : handleMenuClosed()" autoClose="inside" [placement]="['top-start']">
+    <button type="button" class="btn btn-sm py-0" (click)="handleToggle()">
+        <fa-icon [icon]="command.buttonIcon"></fa-icon>
+    </button>
+    <div ngbDropdownAnchor id="anchor" style="position: absolute; width: 0; height: 0" [style.top]="offsetY" [style.left]="offsetX"></div>
```

Comment:
we should avoid inline styles. Move this to the scss file that already exists.

Reply:
Good catch, thank you. I moved it to scss